platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -412,10 +412,31 @@ export default class HangingProtocolService extends PubSubService {
     );
   }
 
+  /**
```

Comment:
Please consider updating the HP service docs pages. https://docs.ohif.org/platform/services/data/HangingProtocolService